barretenberg/cpp/src/barretenberg/ecc/curves/secp256k1/secp256k1.test.cpp

```diff
@@ -423,6 +424,12 @@ TEST(secp256k1, DeriveGenerators)
     }
 }
 
+TEST(secp256k1, CheckPrecomputedGenerators)
```

Comment:
Needed to generate offset generator for secp256k1 for the biggroup general batch mul tests to pass. We're using a specific batch_mul for secp256k1 but it is good to have our general tests also work with secp256k1.